packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md

```diff
@@ -145,6 +147,34 @@ customElements.define('my-element', MyElement);
 {% endswitchable-sample %}
 </pre>
 
+### Asides
+
+You can also insert an aside in your instructions by using the following format:
+
+```html
+<aside class="aside aside-positive">
```

Comment:
Doesn't seem like we need a class `aside` when the element itself is an `aside`? `<aside class="positive">` etc. seems simpler?

Reply:
oh geez it should be alert. Will change though like this syntax more